backstop.json

```diff
@@ -2102,6 +2102,20 @@
       "misMatchThreshold" : 0.1,
       "onBeforeScript": "",
       "onReadyScript": ""
+    },
+    {
+      "label": "input:disabled",
+      "url": "http://localhost:3000/section/1.22.1-2/fullscreen",
+      "hideSelectors": [],
+      "removeSelectors": [],
+      "selectors": [
+        ".object-section"
+      ],
+      "readyEvent": "styleguideRendered",
+      "delay": 20,
+      "misMatchThreshold" : 0.1,
+      "onBeforeScript": "",
+      "onReadyScript": ""
```

Comment:
Given other PRs, this could be reduced to:

```
"label": "input:disabled",
"url": "http://localhost:3000/section/1.22.1-2/fullscreen",
"selectors": [ ".object-section" ],
"delay": 20
```